README.md

```diff
@@ -122,3 +122,32 @@ separately.
 - [Build Artifacts](docs/development/artifacts.md): Documentation about the outputs of the build system.
 - [Releases Page](RELEASES.md): Documentation for how to leverage our build artifacts.
 - [Roadmap for Support](ROADMAP.md): Documentation for our prioritized roadmap to support AMD GPUs.
+
+## Provisioning TheRock 🪨
+
+In order to provision TheRock using either a developer/automated nightly release or a specific CI runner build, use the `build_tool/provision.py` script.
+
+Provisioning script setup:
+
+- `python3 -m venv venv`
+- `source venv/bin/activate`
+- `pip install -r requirements.txt`
+- `python build_tools/provision.py --help`
+
+Examples:
+
+- `python build_tools/provision.py --run-id 14474448215 --amdgpu-family gfx94X-dcgpu`: Downloads the gfx94X S3 artifacts from GitHub CI workflow run 14474448215 to the default output directory `therock-build`
```

Comment:
You could link to https://github.com/ROCm/TheRock/actions/runs/14474448215 for `14474448215` so show where the number comes from. Same for any other example versions/ids.